### ISSUE TITLE
Softmax and CrossEntropyLoss

### ISSUE THREAD

**Issue by siovaneDAZN** [doronharitan/human_activity_recognition_LRCN#4]
Correct me if I'm wrong, but as output function I see you are using a Softmax function and CrossEntropyLoss. As the [doc](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html) says, CrossEntropyLoss already includes the softmax. Wouldn't this cause issues in your loss calculation and hence performances?

**Comment by vuthede** [comment]
Hi @siovaneDAZN , I think you're right! Thanks 
Remove softmax layer in model actually helped the model converge faster.
